DEVELOPMENT.md

```diff
@@ -0,0 +1,35 @@
+# Setting up development environment
+
+## Install the necessary build tools
+You require to have latest stable [Open JDK 8](http://openjdk.java.net/projects/jdk8u/), latest stable 
+[Apache Maven](http://maven.apache.org/) and [Git](http://git-scm.com/) installed on your machine.
+
+## Set up IntelliJ IDEA
+ScaleCube project team uses [IntelliJ IDEA](http://www.jetbrains.com/idea/) as the primary IDE, although we are fine 
+with using other development environments as long as you adhere to our coding style.
+
+### Code style
+ScaleCube project team uses [Google code style](http://google.github.io/styleguide/javaguide.html)
+
+#### Note, in the past, we used an altered version of google code style. We now comply the google style guidelines fully
+
+The best way to go is to format your code *before pushing*
+
+You can use [google-java-format](https://github.com/google/google-java-format).
+There are plugins for [IntelliJ](https://github.com/google/google-java-format#intellij) and [Eclipse](https://github.com/google/google-java-format#eclipse) 
+
+
+## IntelliJ Users (that doesn't want to install the plugin for some reason) should:
+0. Download [this code style configuration](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml)
+Apply the configuration:
+
+## Eclipse Users (that doesn't want to install the plugin for some reason) should:
```

Comment:
missing instruction regards import order in eclipse

Reply:
ok so i will wait for something with after you have clear guideline regards this issue

Reply:
As https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing suggests, I believe a file like this: 
```
0=\#
```
is good enough

Reply:
That's not the default one, it's an empty one.